src/crypto/crypto_context.cc

```diff
@@ -1114,7 +1114,7 @@ int SecureContext::TicketKeyCallback(SSL* ssl,
     return -1;
   }
 
-  argv[2] = env != 0 ? v8::True(env->isolate()) : v8::False(env->isolate());
+  argv[2] = v8::True(env->isolate());
```

Comment:
This was a typo from https://github.com/nodejs/node/commit/f7ed5f4ae3ffe2d7220c36cd2257feb0bea81a91, it should have been `enc != 0`:

```suggestion
  argv[2] = enc != 0 ? v8::True(env->isolate()) : v8::False(env->isolate());
```

Reply:
Ah, that makes much more sense. Thanks for clearing that up @addaleax  :)

Any comments on the changes in src/udp_wrap.cc ?

Reply:
Yeah, those changes look good :+1: